src/main/java/org/codewars/utils/TaskRunner.java

```diff
@@ -3,6 +3,8 @@
 import java.math.BigInteger;
 
 public class TaskRunner {
+    private static final Reader reader = new Reader();
```

Comment:
all methods are static.

Reply:
In class TaskRunner, right?

Reply:
in Reader i mean

Reply:
![image](https://github.com/UA-1023-TAQC/UA-1023-TAQC_CodeWars_Java/assets/50095851/1c46d6b7-32e7-476f-9ba6-012f2ba125f4)
example